include/Core/DB/TableManager.php

```diff
@@ -42,7 +42,7 @@ public function on_activate( $network_wide ) {
 			// Note: if there are more than 10,000 blogs or
 			// if `wp_is_large_network` filter is set, then this may fail.
 			// TODO: Take care of the deprecated function.
```

Comment:
Remove this comment, since you are handling the deprecated function in the commit.